### PR TITLE
Call Transfer - UI Library Sample

### PR DESCRIPTION
**Pull request by mgamis-msft**
## Purpose

Call Composite sample that shows:
- how to setup a listener for the 'transferRequested' event
- how to call a Teams user since the current beta that only Teams users can request transfer right now
- 
## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
```
cd ui-library-transfer
npm i
npm run start 
```
I tested this locally and called a Teams user and tested transfer successfully

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: ui-library-transfer/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+This sample showcases how Call Composites can be used to handle transfer requests. The current beta version can only
+receive transfer requests from Teams users in 1 on 1 calls.
+
+## Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F)  .
+- [Node.js](https://nodejs.org/en/) Active LTS and Maintenance LTS versions (8.11.1 and 10.14.1 recommended).
+- An active Communication Services resource. [Create a Communication Services resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource). You will need the endpoint value for the resource
+- An identity with VoIP. Generate an identity using the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/identity/quick-create-identity).
+- A phone number procured through Azure Communication Service portal using [Azure Portal](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/telephony/get-phone-number).
```
Comment (dmceachernmsft) [inline ui-library-transfer/README.md:21]: Should this be updated about the teams user and some tenant information?

File: ui-library-transfer/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      };
+    } catch (e) {
+      console.error(e);
+      return {};
+    }
+  }, []);
+
+  const afterCallAdapterCreate = useCallback(async (adapter: CallAdapter) => {
+    adapter.on("transferRequested", (e) => {
+      e.accept();
```
Comment (dmceachernmsft) [inline ui-library-transfer/src/App.tsx:42]: In the quick start should we give an example on creating a simple modal for Contoso? or do we want to rely on storybook for that?

File: ui-library-transfer/README.md (hunk 1 above)
Comment (mgamis-msft) [reply ui-library-transfer/README.md]: Thanks. Updated.

File: ui-library-transfer/src/App.tsx (hunk 2 above)
Comment (mgamis-msft) [reply ui-library-transfer/src/App.tsx:42]: I removed the modal example from storybook to avoid the timeout scenario.

**Review by dmceachernmsft — APPROVED** [review]

**Pull request merged**